tests/functional/test_install.py

```diff
@@ -520,25 +520,15 @@ def test_install_editable_from_bazaar(script: PipTestEnvironment) -> None:
 
 
 @pytest.mark.network
```

Comment:
Is network access still necessary with a local installation?

Reply:
Normally installing from source requires internet access to download setuptools, but it turns out the legacy editable install mechanism is being used, thus, pip will use the already provisioned setuptools to build.

This may/will probably break once we remove the legacy editable mechanism, but honestly, I'd like to overhaul pip's end-to-end tests to avoid the network/prefer the local cache as much as possible anyway.